rj_gameplay/rj_gameplay/gameplay_node.py

```diff
@@ -30,7 +30,13 @@
 from rj_gameplay.action.move import Move
 
 # ignore "unused import" error
-from rj_gameplay.play import line_up, basic_defense, keepaway, basic122  # noqa: F401
+from rj_gameplay.play import (  # noqa: F401
+    line_up,  # noqa: F401
+    basic_defense,  # noqa: F401
+    keepaway,  # noqa: F401
+    penalty_defense,  # noqa: F401
+)  # noqa: F401
+from rj_gameplay.play import basic122  # noqa: F401
```

Comment:
What are these noqa: F401 comments?

Reply:
flake8 gives unused import errors. flake8 ignores lines with #noqa F401. Since all plays are tested in the gamplay_node, a bunch of plays is imported.